src/sentry/notifications/notifications/codeowners_auto_sync.py

```diff
@@ -15,6 +15,7 @@
 
 class AutoSyncNotification(ProjectNotification):
     notification_setting_type = NotificationSettingTypes.DEPLOY
+    metrics_key = "auto_sync"
```

Comment:
ultra nit: I believe from the rest of the PR you were organizing these alphabetically, but might have missed this one.